.gitignore

```diff
@@ -112,4 +112,5 @@ src/tutorial/data.json
 draft.md
 
 # vscode specific folders
-.vscode/
\ No newline at end of file
+.vscode/
+/.idea/
```

Comment:
ok aggiungiamo questa riga, capiremo poi in quella ufficiale se per queste modifiche andrà fatto un revert